object_tracking_with_gcp/Object Tracking Tutorial.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
Let's rename folder to object_tracking, and file to object_tracking_with_gcp_pipeline